kuma/settings/common.py

```diff
@@ -1452,3 +1470,23 @@ def get_user_url(user):
 
 # Tell django-taggit to use case-insensitive search for existing tags
 TAGGIT_CASE_INSENSITIVE = True
+
+# Content Experiments
+# Must be kept up to date with PIPELINE_JS setting and the JS client-side
+#  configuration. The 'id' should be a key in PIPELINE_JS, that loads
+#  Traffic Cop and a client-side configuration like
+#  kuma/static/js/experiment-wiki-content.js
+#
+CONTENT_EXPERIMENTS = [{
+    'id': 'experiment-framework-test',
+    'ga_name': 'framework-test',
+    'param': 'v',
+    'pages': [{
+        'locale': 'en-US',
+        'slug': 'Web/JavaScript/Reference/Operators/Comparison_Operators',
+        'variants': [
+            ['control', 50, 'Web/JavaScript/Reference/Operators/Comparison_Operators'],
+            ['test', 50, 'Experiment:FrameworkTest/Comparison_Operators'],
```

Comment:
I alluded to this back in [PR 4146](https://github.com/mozilla/kuma/pull/4146#pullrequestreview-25550272). As a backend dev, it made sense to store all the data in the setting, and then generate some JSON to define the experiment for the front-end.  Instead, you defined the experiment in the .js file, so there's some duplication.  I can remove the percent from here as potentially confusing.